### DISCUSSION TITLE
[new] Add new servers

### DISCUSSION THREAD

**Discussion by KuroRaiken** [aXenDeveloper/ips-app-advanced-serverlist#40] [Q&A] [answered]
Servers:
1. Call Of Duty: Modern Warfare 2
2. Сall Of Duty: Modern Warfare 3
3. Call of Duty: BlackOps
4. S.T.A.L.K.E.R. Call of Pripyat
5. S.T.A.L.K.E.R. Clear Sky
6. S.T.A.L.K.E.R. Shadow of Chernobyl 
7. Space Engineers
8. Team Fortress 2 Classic
9. Team Fortress Classic
10. GTA: Criminal Russia Multiplaye
11. GTA: FiveM
12. GTA: Multi Theft Auto (MTA)
13. GTA: RAGE Multiplayer
14. Half-Life 1 Deathmatch
15. SCP: Secret Laboratory
16. SCUM
17. DEADSIDE
18. WarCraft 3 Battle net (no steam)
[wc3] You can see the warcraft3 server here
[link](http://www.rubattle.net/support/1/) and [link](http://europebattle.net/w3/join)
If you need ip addresses of servers, you can get them on this site ([link](https://www.gs4u.net/ru/games.html)).

If you can add these 2 servers
1. Lineage 2(all server chronicles)
2. World of Warcraft(all server chronicles)
But for these servers, you need to make a connection to the database from the server itself, if necessary, you can make a paid addon)

**Comment by aXenDeveloper** [comment, marked as answer]
The server table will support the custom API from version 2.0.0.
You can add any servers yourself.

**Reply by KuroRaiken** [reply]
> 
Can you show what it looks like via a custom API?
I just don't understand how it works)

**Reply by KuroRaiken** [reply]
> Таблица сервера будет поддерживать пользовательский API, начиная с версии 2.0.0. Вы можете добавить любые серверы самостоятельно.

I don't think the custom API will be able to add L2 servers I WOW without a detailed database connection to the server.

**Reply by aXenDeveloper** [reply]
> > 
> 
> Can you show what it looks like via a custom API? I just don't understand how it works)

You will find out when version 2.0.0 will be released.

**Reply by KuroRaiken** [reply]
> > > 
> > 
> > 
> > Можете ли вы показать, как это выглядит через пользовательский API? Я просто не понимаю, как это работает)
> 
> Вы узнаете, когда выйдет версия 2.0.0.

I ask you very much, can you add servers to the list that are listed above without a custom api function?

**Reply by aXenDeveloper** [reply]
![image](https://user-images.githubusercontent.com/58148176/164692940-0ed3abf2-1a2c-4188-a3ad-bfe630266eec.png)

**Reply by KuroRaiken** [reply]
> ![изображение](https://user-images.githubusercontent.com/58148176/164692940-0ed3abf2-1a2c-4188-a3ad-bfe630266eec.png)

I know about this list, I say add other games to this list that I indicated above in the post)

**Reply by aXenDeveloper** [reply]
You have to create custom API yourself.
See more: https://docs.oracle.com/en/cloud/paas/mobile-autonomous-cloud/get-started-custom-apis/index.html

And for my app:
https://github.com/aXenDeveloper/ips-app-advanced-serverlist#-custom-api

**Reply by KuroRaiken** [reply]
> 



> Вы должны создать собственный API самостоятельно. Подробнее: [https://docs.oracle.com/en/cloud/paas/mobile-autonomous-cloud/get-started-custom-apis/index.html.](https://docs.oracle.com/en/cloud/paas/mobile-autonomous-cloud/get-started-custom-apis/index.html)
> 
> И для моего приложения: https://github.com/aXenDeveloper/ips-app-advanced-serverlist#-custom-api

It's hard for me :(
I hope you yourself will add to the list of servers those games that are not there for people like me, as I don’t understand this API)

**Comment by KuroRaiken** [comment]
I just don't understand because I'm not a programmer :)
I hope you understand us and make life easier for people like us and add games to the list :)

**Comment by KuroRaiken** [comment]
@aXenDeveloper Should I expect you to add new games to the list that will support monitoring?

**Reply by aXenDeveloper** [reply]
Currently no

**Reply by KuroRaiken** [reply]
> В данный момент нет

In which update should we expect 2.3 or 3.0?

**Reply by aXenDeveloper** [reply]
I don't know

**Reply by KuroRaiken** [reply]
> Я не знаю

Некоторые сервера есть в этом приложении https://invisioncommunity.com/files/file/7456-gs-server-list/ 
можете от туда взять :)

**Reply by KuroRaiken** [reply]
> > я не знаю
> 
>Some servers are in this application  https://invisioncommunity.com/files/file/7456-gs-server-list/ you can take from him and add to yours :)

**Comment by aXenDeveloper** [comment]
I testing servers form your list: https://github.com/aXenDeveloper/ips-app-advanced-serverlist/pull/49

**Reply by KuroRaiken** [reply]
> Я тестирую серверы из вашего списка: #49

Cool, thanks, looking forward to the rest :)

**Reply by KuroRaiken** [reply]
If it's not difficult, you can sort the servers correctly and make CUSTOM API 1.

**Reply by KuroRaiken** [reply]
> I testing servers form your list: #49

Any new news?
